test/DynamoCoreTests/SerializationTests.cs

```diff
@@ -346,7 +349,7 @@ private static void SaveWorkspaceComparisonData(WorkspaceComparisonData wcd1, st
 
             var workspaceDataDict = new Dictionary<string, object>();
             workspaceDataDict.Add("nodeData", nodeData);
-            workspaceDataDict.Add("executionDuration", executionDuration);
+            workspaceDataDict.Add("executionDuration", executionDuration.TotalSeconds);
```

Comment:
TotalSeconds.  I thought I wanted TotalMilliseconds, but that's the part before the decimal.  TotalSeconds is whole and decimal seconds, which is exactly what we want.